alaoweb/2021.alaoweb.org
# Display video Captions by default

* closes #180 
* adds captions by default on all poster and session videos
* turns off display of session videos for now (they were showing by default, but no content had been loaded yet)

To test on localhost:
* in config.yml set:
  * `showSessions: true` (line 278) 
  * `showPosterContent: true` (line 207)
* `bundle exec jekyll build` & `bundle exec jekyll serve`
* preview a poster such as: http://localhost:4000/posters/libraries-value-covid-19
* preview a session (right now there's only one with a vid link): http://localhost:4000/schedule/#session-205

The captions should show by default
